app/utilities/metadata_validators.py

```diff
@@ -6,7 +8,7 @@ class RegionCode(validate.Regexp):
     Currently, this does not validate the subdivision, but only checks length
     """
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args: int, **kwargs: Any) -> None:
```

Comment:
Where did this `args` int come from?. This doesn't match the call to the parent.

Reply:
I can't remember exactly why I did that, but that would be incorrect either way and from looking at the parent's method, I believe is should be `Any`?